backend/apps/schemes/services/benefit_service.py

```
@@ -35,63 +68,49 @@ def benefit_create(*, benefit_data: dict, user=None):
+        try:
+            benefit = Benefit.objects.create(**benefit_data)
+            return benefit
+        except ValidationError as e:
+            cls._handle_validation_error(e)
+        except IntegrityError as e:
+            cls._handle_integrity_error(e)
+
+    @classmethod
     @transaction.atomic
-    def benefit_update(*, benefit_id: str, update_data: dict, user=None):
+    def benefit_update(cls, *, benefit_id: str, update_data: dict, user=None):
```

Ruff (0.14.3)

Unused class method argument: `user`

(ARG003)